### PR TITLE
Update pilot-CMAuthority-mde-settings-management-defender.png

### PR DESCRIPTION
**Pull request by msbemba**
updated the screenshot to reflect that boxes are to be unchecked

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/memdocs/issues/3107

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by Brenduns** [comment]
#sign-off

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

File: memdocs/intune/protect/includes/security-config-mgt-prerequisites.md [hunk 1]
```diff
@@ -116,7 +116,8 @@ To support Microsoft Defender for Endpoint security configuration management thr
 
 1. Sign in to [Microsoft 365 Defender portal](https://security.microsoft.com/) and go to **Settings** > **Endpoints** > **Configuration Management** > **Enforcement Scope** and enable the platforms for security settings management:
    :::image type="content" source="../media/mde-security-integration/enable-mde-settings-management-defender.png" alt-text="Enable Microsoft Defender for Endpoint settings management in the Microsoft 365 Defender portal.":::
-1. Configure Pilot Mode and Configuration Manager authority settings to fit your organization needs:
+1. Configure Pilot Mode and Configuration Manager authority settings to fit your organization needs.
```
Comment (JohanFreelancer9) [inline memdocs/intune/protect/includes/security-config-mgt-prerequisites.md:119]:
```suggestion
1. Configure Pilot Mode and Configuration Manager authority settings to fit your organization's needs.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Pull request merged**